src/cljs/rems/fields.cljs

```diff
@@ -142,24 +142,30 @@
   (.. event -target -value))
 
 (defn text-field
```

Comment:
I think it would be really good to add the docstring with all the arguments here.

Reply:
Maybe a better option would be adding it to ``` field-wrapper```?

Reply:
Yes, also I think all the field types should support the info, not just basic text field.